aldryn_django_cms/views.py

```diff
@@ -66,9 +66,13 @@ def check_uninstall_ok(request):
                 installed_apphooks.append(hook)
 
         # generator of classes found in menu.py module
-        cms_menu_classes = get_classes_from_module(app=app, module_name="menu")
+        old_cms_menu_classes = set(get_classes_from_module(app=app, module_name="menu"))
 
-        for menu_class in cms_menu_classes:
+        # 3.4 supports but does not require cms_menus module
+        # will be required in 3.5
+        new_cms_menu_classes = set(get_classes_from_module(app=app, module_name="cms_menus"))
+
+        for menu_class in (old_cms_menu_classes|new_cms_menu_classes):
```

Comment:
Using `set` looses ordering; does this have any influence on how the menus get loaded?